contracts/protocol/libraries/configuration/ReserveConfiguration.sol

```diff
@@ -49,6 +50,7 @@ library ReserveConfiguration {
   uint256 internal constant EMODE_CATEGORY_START_BIT_POSITION = 168;
   uint256 internal constant UNBACKED_MINT_CAP_START_BIT_POSITION = 176;
   uint256 internal constant DEBT_CEILING_START_BIT_POSITION = 212;
+  uint256 internal constant FLASHLOAN_ENABLED_START_BIT_POSITION = 252;
```

Comment:
Unless it was previously used for other stuff, ewe can use the reserved 63th bit

Reply:
Not sure if this bit is reserved for something or has been used previously. @The-3D any insight?